mirumee/saleor
# Automatic plugin discovery

This enables plugins to be discovered automatically once they are installed. It defines a `setuptools` entry point for `saleor.plugins`.

How to test:

1. Install the example plugin:
    ```shell
    pip install git+https://github.com/patrys/saleor-example-plugin.git
    ```
2. Visit your dashboard to confirm that the plugin is installed.

# Impact

* [x] Docs need to be updated

# Pull Request Checklist



* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database migration files are up to date
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog
